substrate/frame/revive/src/gas.rs

```diff
@@ -219,6 +219,35 @@ impl<T: Config> GasMeter<T> {
 		Ok(ChargedAmount(amount))
 	}
 
+	/// Charge the initial cost for executing EVM bytecode.
+	pub fn charge_evm_init_cost(&mut self) -> Result<(), DispatchError> {
+		self.gas_left = self
+			.gas_left
+			.checked_sub(&T::WeightInfo::evm_opcode(0))
+			.ok_or_else(|| Error::<T>::OutOfGas)?;
+		Ok(())
+	}
```

Comment:
We don't really do that for PVM. The costs are either included in:

1) The `call/instantiate` extrinsic.
2) The `call/instantiate` syscall.

Doing this explicitly seems like double charging.